### PR TITLE
Adding link to more info about Classic WF retirement

### PR DESCRIPTION
**Pull request by get-itips**
Closes https://github.com/MicrosoftDocs/OfficeDocs-SharePoint/issues/2269
Closes https://github.com/MicrosoftDocs/OfficeDocs-SharePoint/issues/2318

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: migration/sp-teams-sites-migration-guide.md [hunk 1]
```diff
@@ -87,7 +87,7 @@ To learn more:
 - [Get started with Power Automate](https://docs.microsoft.com/flow/getting-started)
 
 >[!Note]
```
Comment (JohanFreelancer9) [inline migration/sp-teams-sites-migration-guide.md:89]:
```suggestion
> [!NOTE]
```

File: migration/sp-teams-sites-migration-guide.md [hunk 2]
```diff
@@ -87,7 +87,7 @@ To learn more:
...
->Classic workflow will be supported and available until 2026. We recommend taking this into consideration as you plan for your workflow lifetime.
+>Classic workflow will be supported and available until 2026. We recommend taking this into consideration as you plan for your workflow lifetime. For more information, please see [SharePoint 2010 workflow retirement](https://support.microsoft.com/office/sharepoint-2010-workflow-retirement-1ca3fff8-9985-410a-85aa-8120f626965f).
```
Comment (JohanFreelancer9) [inline migration/sp-teams-sites-migration-guide.md:90]:
```suggestion
> Classic workflow will be supported and available until 2026. We recommend taking this into consideration as you plan for your workflow lifetime. For more information, please see [SharePoint 2010 workflow retirement](https://support.microsoft.com/office/sharepoint-2010-workflow-retirement-1ca3fff8-9985-410a-85aa-8120f626965f).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@JoanneHendrickson PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by get-itips** [comment]
@JoanneHendrickson Can you kindly review and merge? Thanks!

**Comment by get-itips** [comment]
@JoanneHendrickson can you kindly review? Thanks

**Pull request merged**